gusb/gusb-device.c

```diff
@@ -205,16 +205,10 @@ g_usb_device_init (GUsbDevice *device)
 static void
 g_usb_device_build_platform_id_cb (GString *str, libusb_device *dev)
```

Comment:
at this point maybe it makes sense to rename the method to some more clear like device_get_parent_port_number or something like that

Reply:
I think we're stuck with the name for now. I'd be happy to add better gtk-doc comment text if you can think of something.

Reply:
why? the method is static and not exposed no?

Reply:
Ohh, the static one! Sure, can do, sorry. One sec.

Reply:
Done.